.github/workflows/validations.yaml

```diff
@@ -29,7 +29,8 @@ jobs:
   Unit-Test:
     # Note: changing this job name requires making the same update in the .github/workflows/release.yaml pipeline
     name: "Unit tests"
-    runs-on: ubuntu-24.04
+    # we need more storage than what's on the default runner
+    runs-on: ubuntu-22.04-4core-16gb
```

Comment:
Is this concerning? Is there something else we should do to limit/reduce the overall test resources used?

Reply:
It looks like tests didn't kick off the last commit on this PR, I hope this isn't due to some resource constraints on the larger runners.